### DISCUSSION TITLE
Thoughts on glassworking

### DISCUSSION THREAD

**Discussion by jeremyshannon** [jeremyshannon/Exile#18] [General]
Quick thoughts on the new glass working system:
- Panes, pane casting trays, and glass ingots should all be removable by hand
- Window facing is weird and doesn't always place in the direction it seems like it should?
- Maybe windows should be removable by hand, too? That could let  dirty stone-age peasants break into your patrician iron-age brick/antiquorium home if we ever add locks to doors.
- Maybe the glass furnace should be a brickworking craft instead of using the chopping block to create it? Chopping block seems odd for an assembly of bricks. OTOH, brickworking has a lot of recipes
- I think the glass furnace should need to be heated by a proper furnace adjacent to it. It could perhaps turn into a crafting table which is just a glowing version of itself, when hot enough

- Potash node timers only run once! If you place the potash solution pot down too soon before your furnace gets hot, you won't get potash until you pick it up and put it back down. It needs to restart on a fail.

All in all, this is a great addition!

**Comment by wizardofgcc** [comment]
Thanks for the feedback! Most of these are bugs or dumb mistakes, so I'm going to fix them. The glass furnace needing actual heat is an interesting idea, would be good to try it out.
